devops/developer-api/openapi.yml

```diff
@@ -0,0 +1,1843 @@
+          format: int64
+        db_lua_credits_per_min:
+          type: integer
+          description: Lua script execution credits per minute
+          format: int64
+        db_store_max_idle:
+          type: integer
+          description: Store connection idle timeout in nanoseconds
+          format: int64
+        db_max_loads_per_sec:
+          type: integer
+          description: Maximum load operations per second
+          format: int64
+        db_acl_enabled:
+          type: boolean
+          description: Access Control List is enabled
+        db_acl_default_user_status:
+          type: boolean
+          description: Default user access status in ACL
+        db_eviction:
+          type: boolean
+          description: Database-level eviction policy status
+      xml:
+        name: database
+    CreateDatabaseRequest:
+      type: object
+      properties:
+        name:
+          type: string
+          description: Name of the database
+          example: "myredis"
+        region:
+          type: string
+          description: Region of the database. Only valid option is 'global'.
+          enum:
+            - global
+          default: "global"
+          example: "global"
+        primary_region:
+          type: string
+          description: Primary Region of the Global Database
+          enum:
+            - us-east-1
+            - us-west-1
+            - us-west-2
+            - eu-west-1
+            - eu-central-1
+            - ap-southeast-1
+            - ap-southeast-2
+            - sa-east-1
+          example: "us-east-1"
+        read_regions:
+          type: array
+          items:
+            type: string
+            enum:
+              - us-east-1
+              - us-west-1
+              - us-west-2
+              - eu-west-1
+              - eu-central-1
+              - ap-southeast-1
+              - ap-southeast-2
+              - ap-northeast-1
+              - sa-east-1
+          description: Array of Read Regions of the Database
+          example: ["us-west-1", "us-west-2"]
+        plan:
+          type: string
+          description: |
+            Specifies the fixed plan type for the database.
+            If omitted, the database defaults to either the pay-as-you-go or free plan, based on the account type.
+          enum:
+            - payg
+            - fixed_250mb
+            - fixed_1gb
+            - fixed_5gb
+            - fixed_10gb
+            - fixed_50gb
+            - fixed_100gb
+            - fixed_500gb
+          example: "payg"
+        budget:
+          type: integer
+          description: Monthly budget of the database
+          example: 100
+      required:
+        - name
+        - region
+        - primary_region
+
+    ChangePlanRequest:
+      type: object
+      properties:
+        plan_name:
+          type: string
+          description: The new plan for the database
+          enum:
+            - payg
```

Comment:
also missing free. Is there no way to define these enums in somewhere common and use it everywhere else?